lib/portlayer/exec/container.go

```diff
@@ -351,6 +362,28 @@ func (c *Container) start(ctx context.Context) error {
 
 	err := c.containerBase.start(ctx)
 	if err != nil {
+		// change state to stopped because start task failed
+		c.SetState(StateStopped)
+
+		// check if locked disk error
+		devices := disk.LockedDisks(err)
+		if len(devices) > 0 {
+			for i := range devices {
+				// get device id from datastore file path
+				// FIXME: find a reasonable way to get device ID from datastore path in exec
+				devices[i] = strings.TrimSuffix(path.Base(devices[i]), ".vmdk")
```

Comment:
hmm.. good question. Might not, but if that's file locked issue, in our case, most likely it's vmdk file attached to multiple VMs.
I didn't see other scenario for that. 

But no matter how, this is just the best effort we can do right now to get the device ID instead of device path, cause the device path does not mean anything for end user. 
If there's any other file conflict happened, it would be better to keep the file suffix, I think